platform/iphone/detect.py

```diff
@@ -167,38 +167,6 @@ def configure(env):
         LINKFLAGS=[
             "-isysroot",
             "$IPHONESDK",
-            "-framework",
-            "AudioToolbox",
-            "-framework",
-            "AVFoundation",
-            "-framework",
-            "CoreAudio",
-            "-framework",
-            "CoreGraphics",
-            "-framework",
-            "CoreMedia",
-            "-framework",
-            "CoreVideo",
-            "-framework",
-            "CoreMotion",
-            "-framework",
-            "Foundation",
-            "-framework",
-            "GameController",
-            "-framework",
-            "MediaPlayer",
-            "-framework",
-            "OpenGLES",
```

Comment:
Isn't this still relevant to keep? Or is it added somewhere else?

Reply:
Since iOS export was previously updated to support modules with `-fmodules` and `-fcxx-modules` flags there should be no need to manually set system frameworks for linkage. Should keep build instructions more clear.
But I can revert this change if keeping them is better

Reply:
>  But I can revert this change if keeping them is better

No I trust you there, I don't have experience with this. I just asked in case this was overlooked among other frameworks which might have been needed for the plugins only.